clang/tools/3c/utils/port_tools/generate_ccommands.py

```diff
@@ -190,7 +190,12 @@ def run3C(checkedc_bin, compile_commands_json, checkedc_include_dir, skip_paths,
     args = []
     args.append(prog_name)
     args.extend(DEFAULT_ARGS)
-    args.extend(list(set(total_x_args)))
```

Comment:
Why is it ok delete `total_x_args`. Do the argument still make it into the arg list through `s`?

Reply:
The idea was that `3c` would get the arguments directly from the compilation database specified via `-p`.  But I haven't tested this carefully and the current design hasn't caused a problem yet, so I decided it's better to revert the change from this PR and make it later if desired.